modules/swagger-codegen/src/main/resources/JavaSpring/beanValidation.mustache

```diff
@@ -1,5 +1,6 @@
 {{#required}}
   @NotNull
-{{/required}}
+{{/required}}{{#isContainer}}{{^isPrimitiveType}}{{^isEnum}}
```

Comment:
It's Hibernate Validator, which is an implementation of the Bean Validation.  Spring uses it for validation.  It is not to be mixed with Hibernate ORM, which is the persistence library.